### ISSUE TITLE
Docs: Add creation of conf-folder

### ISSUE THREAD

**Issue by laenan8466** [pawamoy/pypi-insiders#5]
With both, Windows 10 (PS) and Linux (Debian/WSL) I ran in some trouble, using the [Installation script](https://pawamoy.github.io/pypi-insiders/#installation).

After installation and before adding the repo in the subprocess, I needed to create the configuration folder.
For Windows: `'C:\\Users\\USERNAME\\AppData\\Local\\pypi-insiders\\pypi-insiders'` 
For Linux(Debian/WSL): `/home/USERNAME/.config/pypi-insiders`

Maybe this has to be added to documentation.

For Windows I was able to get the temp server running, but installation from pipx was not working (could be bc. of conflicting localhost from WSL...). Managed it with the localhost-server from WSL.

Further note:
Updating with `pypi-insiders update` was not possible in Windows - CI is testing for windows. Is it useful to create a new issue for this or 'will not fix'? For my personal use, WSL/Linux is good enough/prefered.

**Comment by pawamoy** [comment]
Thanks for the report!

> After installation and before adding the repo in the subprocess, I needed to create the configuration folder.
For Windows: 'C:\\Users\\USERNAME\\AppData\\Local\\pypi-insiders\\pypi-insiders'
For Linux(Debian/WSL): /home/USERNAME/.config/pypi-insiders

We should definitely create this folder if it does not yet exist when running the program.

> For Windows I was able to get the temp server running, but installation from pipx was not working (could be bc. of conflicting localhost from WSL...). Managed it with the localhost-server from WSL.

I'll need more info to investigate. Feel free to open a new issue :slightly_smiling_face: 

> Updating with pypi-insiders update was not possible in Windows - CI is testing for windows. Is it useful to create a new issue for this or 'will not fix'?

A new issue detailing the problem would be greatly appreciated, yes!

**Comment by pawamoy** [comment]
> We should definitely create this folder if it does not yet exist when running the program.

Actually we already do this :thinking: Any command reading/writing the repositories configuration will create the folder:

```python
        self.conf_path: Path = Path(conf_path)
        self.conf_path.parent.mkdir(exist_ok=True)
```

I think it simply needs `parents=True` too.

**Comment by pawamoy** [comment]
Should be fixed in 0.0.0.1.3.2!
